pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -754,4 +753,33 @@ in rec {
     readmeFile = "README_sk.txt";
     license = with stdenv.lib.licenses; [ gpl2 lgpl21 mpl11 ];
   };
+
+  /* DANISH */
+
+  da_DK = da-dk;
+  da-dk = mkDict rec {
+    name = "hunspell-dict-da-dk-${version}";
+    version = "2.5.137";
+
+    src = fetchurl {
+      url = "https://stavekontrolden.dk/dictionaries/da_DK/da_DK-${version}.oxt";
+      sha256 = "16y0smkg1mq0133r1fbw5ak6s2xw39281knk5ivhanakayq789qx";
+    };
+
+    shortName ='da-dk';
+    shortDescription = 'Danish (Danmark)';
+    dictFileName = "da_DK";
+    readmeFile = "README_da_DK.txt";
+    nativeBuildInputs = [ unzip ];
+    unpackCmd = ''
+      unzip $src ${dictFileName}.dic ${dictFileName}.aff ${readmeFile} -d ${dictFileName}
+    '';
+
+    meta = with stdenv.lib; {
+      description = "Hunspell dictionary for Danish (Denmark) from Stavekontrolden";
+      homepage = "https://stavekontrolden.dk";
+      license = with stdenv.lib.licenses; [ gpl2 lgpl21 mpl11 ];
```

Comment:
https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206

Reply:
Fixed. It's GPL3Only according to it's git repo

Reply:
Looks like the previous versions were correct according to https://github.com/jeppebundsgaard/stavekontrolden/blob/c41925e71bde3d68cbcb27b64b9e2f521516040f/dictionaries/da_DK/oxt-files/README_da_DK.txt, just needed `Only` appended.

Reply:
You're right. Only checked the license file in the first place.